templates/_components/free_text.html.twig

```diff
@@ -0,0 +1,21 @@
+<div class="govuk-width-container">
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-half" >
+            <div class="govuk-border-right-free-text" style="height:110px ; margin-left:20px "/>
+            <div>{{ component.free_text_free_text_left_content_banner_new | raw}}</div>
+             <div style="margin-top: 50px ; font-size:2px; line-height: 3px ; margin-right:8px">{{ component.free_text_free_text_left_content_new| raw}}</div>
+        {% if component.free_text_free_text_cta_label|trim is not empty and component.free_text_free_text_cta_destination|trim is not empty %}
+            <a href="{{ component.free_text_free_text_cta_destination }}" class="govuk-button">{{ component.free_text_free_text_cta_label }}</a>
+        {% endif %}
+
+        </div>
+    </div>
+    
+    
+    <div class="govuk-grid-column-one-half">
+        <div style="margin-right:31px"> {{component.free_text_free_text_right_content_banner_new | raw}}</>
```

Comment:
There are a few syntax errors in this file (for example at the end of this line the closing tag is malformed).

Mohammed has fixed these in a separate branch, so I don't think it's necessary to fix them here again (it will only cause merge conflicts), but I would like to go through the problems with this file with both if you if possible.